examples/1-hello-world/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  nixpkgs ? builtins.fetchTarball {
```

Comment:
Recommend commenting that these are just pins (looks hairy for a hello world) or factoring out to pins.nix or something.

Reply:
That might be a good idea, sure. Maybe we could use a common tool like Niv to simplify the `default.nix`, perhaps?